<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vo Minh Thanh</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>411</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>411</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAge">
       <property name="text">
        <string>Age:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButtonNew">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>New</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image/ic_new.png</normaloff>image/ic_new.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelEmployeeID">
       <property name="text">
        <string>Employee ID:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButtonSave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image/ic_save.png</normaloff>image/ic_save.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButtonDelete">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image/ic_delete.png</normaloff>image/ic_delete.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditAge"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditID"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>449</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>System</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExport_to_JSon"/>
    <addaction name="actionImport_from_JSon"/>
    <addaction name="separator"/>
    <addaction name="actionExir"/>
   </widget>
   <addaction name="menuSystem"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport_to_JSon">
   <property name="icon">
    <iconset>
     <normaloff>image/ic_export.png</normaloff>image/ic_export.png</iconset>
   </property>
   <property name="text">
    <string>Export to JSon</string>
   </property>
  </action>
  <action name="actionImport_from_JSon">
   <property name="icon">
    <iconset>
     <normaloff>image/ic_importdatabase.png</normaloff>image/ic_importdatabase.png</iconset>
   </property>
   <property name="text">
    <string>Import from JSon</string>
   </property>
  </action>
  <action name="actionExir">
   <property name="icon">
    <iconset>
     <normaloff>image/ic_exit.png</normaloff>image/ic_exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
